openai/openai.go

```diff
@@ -35,11 +39,11 @@ const (
 	`
```

Comment:
oac.client should be initialized within the constructor of the OpenAiClient struct instead of in the main function.

Reply:
oh cool, wasn't aware that Go had a constructor. fixed.